doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,109 @@
+# How we Improved the Files Virtual Schema Performance
+
+Our [Virtual Schemas for document files](https://github.com/exasol/virtual-schema-common-document-files) allow our customer to integrate data from files into their Exasol database. Customers use this feature for integrating external data on demand into their analytics and for building ETL pipelines.
+
+For Exasol speed matters. So does it for our integrations. For customers its crucial that the queries that include external data run as fast as possible. In this blog post I want to give some insight on the process of our recent performance improvements of the Virtual Schema for parquet files.
+
+## How it Begins
+
+For me this project started when our product manager asked me to improve the performance of the parquet file loading. He compared the performance of our products with competitors and found out, that parquet loading needs some improvements.
+
+In order to do professional performance improvements we use the simple workflow:
+
+Measure -> Improve -> Measure.
+
+In my eyes, this simple strategy brings lots ot advantages:  
+First it allows you to check weather a change helped or if it even made some functionality slower. For example improving the performance of loading big parquet files can easily slow down the loading of many small files. Second, this strategy allows you to define a goal when to stop. Otherwise, you will keep improving forever since there's always some space for improvements.
```

Comment:
```suggestion
First, it allows you to check whether a change helped or if it even made some functionality slower. For example improving the performance of loading big Parquet files can easily slow down the loading of many small files. Second, this strategy allows you to define a goal when to stop. Otherwise, you will keep improving forever since there's always a space for improvements.
```